lib/ansible/modules/network/ldap_entry.py

```diff
@@ -128,6 +134,15 @@
     bind_dn: cn=admin,dc=example,dc=com
     bind_pw: password
 
+- name: Check if Entry exists
+  ldap_entry:
+    dn: cn=users,cn=compat,dc=example,dc=com
+    state: present
+    server_uri: ldaps://localhost
+    bind_dn: cn=Directory Manager
+    bind_pw: password
+    objectClass: organizationalUnit
+    validate_certs: True
```

Comment:
This line doesn't make much sense if the default value is `True`. The whole example is redundant. Please remove it.